src/datasources/alerts-api/configuration/alerts-api.configuration.ts

```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
Nit: can we add a comment why this is necessary?

Reply:
Not needed. Leftover from local testing.

[3be66bc](https://github.com/safe-global/safe-client-gateway/pull/1372/commits/3be66bcdefeb534bec04fbec96acf7397dca4a5d)